src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -238,85 +232,39 @@ class HeaderUI extends Component<Props, State> {
-            flyoutIsCollapsed={this.state.flyoutIsCollapsed}
-            flyoutIsAnimating={this.state.flyoutIsAnimating}
-            onMouseEnter={this.expandDrawer}
-            onFocus={this.expandDrawer}
-            onBlur={this.focusOut}
-            onMouseLeave={this.collapseDrawer}
-            mobileIsHidden={this.state.mobileIsHidden}
-            showScrollbar={this.state.showScrollbar}
-            data-test-subj={classNames(
-              'navDrawer',
-              this.state.flyoutIsAnimating
-                ? null
-                : this.state.isCollapsed
-                ? 'collapsed'
-                : 'expanded'
-            )}
-          >
-            <EuiNavDrawerMenu id="navDrawerMenu" onClick={this.onNavClick}>
-              <EuiListGroup>
-                <EuiListGroupItem
-                  label="Recently viewed"
-                  iconType="clock"
-                  size="s"
-                  style={{ color: 'inherit' }}
-                  aria-label="Recently viewed items"
-                  onClick={() => this.expandFlyout()}
-                  isDisabled={recentlyAccessed.length > 0 ? false : true}
-                  extraAction={{
-                    color: 'subdued',
-                    iconType: 'arrowRight',
-                    iconSize: 's',
-                    'aria-label': 'Expand to view recent apps and objects',
-                    onClick: () => this.expandFlyout(),
-                    alwaysShow: true,
-                  }}
-                />
-              </EuiListGroup>
-              <EuiHorizontalRule margin="none" />
-              <EuiListGroup data-test-subj="appsMenu">
-                {navLinks.map(navLink =>
-                  navLink.hidden ? null : (
-                    <EuiListGroupItem
-                      key={navLink.id}
-                      label={navLink.title}
-                      href={navLink.href}
-                      iconType={navLink.euiIconType}
-                      size="s"
-                      style={{ color: 'inherit' }}
-                      aria-label={navLink.title}
-                      isActive={navLink.active}
-                      data-test-subj="appLink"
-                    />
-                  )
-                )}
-              </EuiListGroup>
-            </EuiNavDrawerMenu>
-            <EuiNavDrawerFlyout
-              id="navDrawerFlyout"
-              title="Recent items"
-              isCollapsed={this.state.flyoutIsCollapsed}
-              listItems={recentlyAccessed.map(item => ({
-                label: item.label,
-                href: item.href,
-                iconType: item.euiIconType,
-                size: 's',
-                style: { color: 'inherit' },
-                'aria-label': item.label,
-              }))}
-              onMouseLeave={this.collapseFlyout}
-              wrapText={true}
+        <EuiNavDrawer ref={this.setNavDrawerRef} data-test-subj={classNames('navDrawer')}>
```

Comment:
Let's be a little more specific with the class name here. We need a prefix. I see both `kbn` and `chr` (for 'chrome') used in this folder. We should probably, formally, decide how we want to handle these kibana core/chrome folders and their class prefixes. @snide thoughts?

Reply:
I have removed the className bit from here, now it aligns with the other data-test-subj's in this file... navDrawer, navDrawerAppsMenu, navDrawerAppsMenuLink

There used to be some conditional classes on this that are no longer possible, so this was not exactly a CSS class, but used solely as an id for the test. Point being, I'm assuming then that the `kbn` vs `chr` thing is now moot.

Reply:
Since there is no logic around it, `classNames('navDrawer')` will literally just spit out `navDrawer` so I think you can change that to: `data-test-subj="navDrawer"`